src/library/tools/R/license.R

```diff
@@ -611,6 +614,8 @@ function(x)
                                stringsAsFactors = FALSE)
         }
 
+        sapply(expansions, `[[`, "SPDX") |>
```

Comment:
Some R developers try to avoid `sapply` because the simplify parameter can break further steps. Not sure if it is better to use `lapply` and then `unlist`.

Reply:
Given that the SPDX field will always be a string, we could replace it with
```suggestion
        vapply(expansions, FUN.VALUE = character(1), FUN = function(x) {x[["SPDX"]]}) |>
```